### DISCUSSION TITLE
Info BigEvent

### DISCUSSION THREAD

**Discussion by Easygamer01s** [Crownicles/Crownicles#1837] [Suggestions fermées]
Bonjour ! 

L'année passée, je me suis permis de reprendre les fichiers JSON indiquant les events, réactions et possibilités. Pour la combinaisons de ces trois variables, j'ai repris les états, malus de temps, de vie et gain en or, xp et si drop. J'ai obtenu un fichier xls d'environ 700 lignes. J'ai aussi générer une matrice avec les temps de trajets entre chaque destinations.

Après la préparation de données, j'ai eu beaucoup de plaisir à analyser chaque event et d'essayer de faire émerger une stratégie simple qui convenait le mieux à mes aspirations dans le jeu.

Cette extraction n'est plus possible car il me semble que certaines informations ci-dessus ne sont plus accessibles directement dans les fichiers. Ne pourrait-on pas voir ces données dans le guide du bot ? Je pense que cela permettrait à d'autres joueurs de pouvoir aussi prendre du plaisir à analyser ces données et de faire émerger une stratégie de jeu personnelle..

Qu'en pensez-vous ? =D

**Comment by BastLast** [comment]
Hello ! En théorie c'est toujours possible à faire, on a pas caché de données 🤔

Le guide du bot cependant n'a, à mon avis pas vocation à être aussi poussé, je pense qu'une partie du fun des gens qui veulent aller aussi loin est de réaliser cette analyse par eux même.

On reste dispo si ya des questions cependant ☺️

**Comment by Easygamer01s** [comment]
J'y ai retrouvé la majorité des informations, mais pas le gain en xp =P
Probablement qu'en remontant dans certaines commandes je devrais pouvoir localiser l'info ..

Merci pour ta réponse !

**Reply by BastLast** [reply]
le gain en xp il est calculé et non plus enregistré

**Reply by Easygamer01s** [reply]
Merci pour l'info =D
